### ISSUE TITLE
Why two guards related to K8S: use-kubernetes and use-local-kubernetes

### ISSUE THREAD

**Issue by agoncal** [quarkusio/quarkus-workshops#397]
In the documentation we have two guards refering to K8s and I can't see why there is a difference. Are we talking about Minikube vs K8s ?

**Comment by holly-cummins** [comment]
No, that sounds like a mistake by me :) I know one is `use-kubernetes`, what's the other one?

**Comment by holly-cummins** [comment]
I think they're slightly different which is why we have two, but we should review to see if they can be consolidated a bit. One was for the case where someone wants to do all the kube stuff but has to install minkube/rancher/etc locally to make progress, and the other was the more general 'we will do kube stuff.' If it's something like an openshift workshop, we want the remote kube content, but people won't have any need to install rancher or minikube. So I think that's why we have two.
